AppCenterDataStorage/AppCenterDataStorageTests/MSDBDocumentStoreTests.m

```diff
@@ -16,14 +27,131 @@ @implementation MSDBDocumentStoreTests
 
 - (void)setUp {
   [super setUp];
+  [MSUtility deleteItemForPathComponent:kMSDBDocumentFileName];
   self.sut = [MSDBDocumentStore new];
-  self.dbStorageMock = OCMClassMock([MSDBStorage class]);
-  self.sut.dbStorage = self.dbStorageMock;
 }
 
 - (void)tearDown {
+  [MSUtility deleteItemForPathComponent:kMSDBDocumentFileName];
   [super tearDown];
-  [self.dbStorageMock stopMocking];
+}
+
+- (void)testReadUserDocumentFromLocalDatabase {
+
+  // If
+  NSString *documentId = @"12829";
+  NSString *partitionKey = @"user";
```

Comment:
Those values are used multiple several times; I think those can be moved out of methods to either constants or setUp procedure.

Reply:
Probably fine for unit tests since they are actually arbitrary strings